src/content/basics/azure-architecture/index.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "The Giant Swarm Azure Architecture"
+description = "Architecture Overview showing how Giant Swarm is set up on Microsoft Azure"
+date = "2019-07-16"
+weight = 20
+type = "page"
+categories = ["basics"]
++++
+
+# The Giant Swarm Azure Architecture
+
+Giant Swarm's Architecture is split into two logical parts, one being the Control Plane and the other being multiple Tenant Clusters. Among other services (for details see Service Architecture below), the control plane runs our [Azure Operator](https://github.com/giantswarm/azure-operator), which handles the full lifecycle management of Tenant Clusters.
+
+## Giant Swarm control plane
+
+![Control Plane Architecture](/img/architecture-azure-control-plane.png)
+
+The above diagram shows our control plane setup running with two bastion hosts in different availability zones. All instances are in a private subnet and only accessible through the bastion hosts. Bastions hosts are available via Azure VPN connection only.
```

Comment:
> The above diagram shows our control plane setup running with two bastion hosts in different availability zones.

I'm not sure about the AZ part here. @kopiczko can you help with this?

Reply:
AFAICT we don't use AZs in CP. We only use availability sets (AS) which according to what I linked here is something like separate rack. We use 3 fault domains. Masters, workers and bastions are in separate ASs. https://cloudopszone.com/availability-sets-and-availability-zones/

For TCs I think we don't even use availability sets.

Reply:
Thanks for this. The control plane diagram got updated to show the fault domains. The tenant cluster doesn't have this but neither does the AWS diagram. So I think that's fine for the first version.

Reply:
The image may suggest that the bastions/masters/workers are collocated in the same fault domains which is not true. I don't know if this is intentional compromise to make the image clearer. Otherwise LGTM.

Reply:
Yes it makes the image clearer. Going to go with this for the first version.